supertokens_python/recipe/emailpassword/types.py

```diff
@@ -55,3 +58,23 @@ def __init__(self, id: str, validate: Callable[[  # pylint: disable=redefined-bu
         self.id = id
         self.validate = validate
         self.optional = optional
+
+
+_T = TypeVar('_T')
+
+# class EmailDeliveryInterface(ABC, Generic[_T]):
+#     pass
+
+
+class TypeEmailPasswordPasswordResetEmailDeliveryInput:
+    def __init__(self, user: User, password_reset_link: str, user_context: Dict[str, Any]) -> None:
```

Comment:
User type will have to change

Reply:
Created `TypeEmailPasswordPasswordResetEmailDeliveryInputUser`